docs/src/pages/docs/introduction.mdx

```diff
@@ -10,20 +10,30 @@ is fast, lightweight, intuitive, and loaded with options that make it easy for u
 sets to explore and visualize their data, from simple pie charts to highly detailed deck.gl
 geospatial charts.
 
-Here's an overview of the key features of Superset:
-
-- A rich set of data visualizations out of the box
-- An easy-to-use interface for exploring and visualizing data
-- Create and share dashboards
-- Enterprise-ready authentication with integration with major authentication providers (database,
-  OpenID, LDAP, OAuth & REMOTE_USER through Flask AppBuilder)
-- An extensible, high-granularity security/permission model allowing intricate rules on who can
-  access individual features and the dataset
-- A simple semantic layer, allowing users to control how data sources are displayed in the UI by
-  defining which fields should show up in which drop-down and which aggregation and function metrics
-  are made available to the user
-- Integration with most SQL-speaking RDBMS through SQLAlchemy
-- Deep integration with Druid.io
+Here are a **few different ways you can get started with Superset**:
+
+- Download the [source from Apache Foundation's website](https://dist.apache.org/repos/dist/release/superset/1.0.0/)
+- Download the latest Superset version from [Pypi here](https://pypi.org/project/apache-superset/)
+- Setup Superset locally with one command
+using [Docker Compose](docs/installation/installing-superset-using-docker-compose)
+- Download the [Docker image](https://hub.docker.com/r/apache/incubator-superset) from Dockerhub
```

Comment:
do we want to point to `https://hub.docker.com/r/apache/superset` now?

Reply:
good catch, fixing now!